IGHTMLQuery/Classes/IGXMLNodeQuery.h

```diff
@@ -19,7 +19,7 @@
  @param xpath used to query the document
  @return elements matched by supplied XPath query.
  */
-- (IGXMLNodeSet*) queryWithXPath:(NSString*)xpath;
+- (nullable IGXMLNodeSet*) queryWithXPath:(nullable NSString*)xpath;
```

Comment:
OK, so we're in agreement. :-)  Mind you, I just didn't want to take TOO many liberties here -- this is your project, so I wanted to err on the conservative side. But this is fair and good. I'll work to make these changes.